src/synth/fluid_synth.c

```diff
@@ -439,18 +441,12 @@ fluid_synth_init(void)
 
 static FLUID_INLINE unsigned int fluid_synth_get_ticks(fluid_synth_t* synth)
 {
-  if (synth->eventhandler->is_threadsafe)
-    return fluid_atomic_int_get(&synth->ticks_since_start);
-  else
-    return synth->ticks_since_start;
```

Comment:
Too early. Will be addressed in #197.

Reply:
again, atomics type need to be opaque, i didnt want to get rid of is_threadsafe, but I need a fuction to access the atomic